### PR TITLE
nisra csh content changes

### PR DESCRIPTION
**Pull request by lauraedwards1**
### What is the context of this PR?
Content changes needed for nir

### How to review
CSH changes on google sheet attached to [trello card](https://trello.com/c/bdKuIEkn) 
[Quick Launcher](https://staging-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-csh-changes/schemas/en/census_individual_gb_nir.json)

### Checklist

* [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### Schemas Artifacts
Schemas artifacts are available at:
```bash
https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/<branch_name>/schemas/en/<schema_name>.json
```

### CODE REVIEW

File: source/jsonnet/northern-ireland/individual/blocks/employment/job_pending.jsonnet [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
   title: title,
   type: 'General',
   answers: [
     {
+      guidance: {
+        show_guidance: guidanceHeader,
+        hide_guidance: guidanceHeader,
+        contents: [
+          {
+            description: 'To get a true picture of the UK working population, we ask this question of everyone who is not currently working. We ask people who are retired because the number of people continuing to work after retirement age is increasing. We ask people who are long-term sick or disabled because some intend to go back to work.',
```
Comment (berroar) [inline source/jsonnet/northern-ireland/individual/blocks/employment/job_pending.jsonnet:15]: Missing a change to the last sentence of the description here too.

File: source/jsonnet/northern-ireland/individual/blocks/employment/main_employment_block.jsonnet [hunk 2]
```diff
@@ -10,7 +10,10 @@ local rules = import 'rules.libsonnet';
         title: 'Main job',
         contents: [
           {
-            description: 'The next set of questions is about your main job. Your main job is the job in which you usually work the most hours.',
+            description: 'The next set of questions is about your last main job. This is the most recent job you had.',
```
Comment (berroar) [inline source/jsonnet/northern-ireland/individual/blocks/employment/main_employment_block.jsonnet:13]: Might want to double check with @donnap573 but having checked the spreadsheet this change looks like it's supposed to be for the past variant?

File: source/jsonnet/northern-ireland/individual/blocks/identity-and-health/language.jsonnet [hunk 3]
```diff
@@ -17,7 +17,7 @@ local question(title, definitionDescription) = {
   title: title,
   type: 'General',
   definitions: [{
-    title: 'What do we mean by “main language”?',
+    title: 'What we mean by “main language”?',
```
Comment (petechd) [inline source/jsonnet/northern-ireland/individual/blocks/identity-and-health/language.jsonnet:20]: There shouldn't be any question mark in the new version.

**Comment by LJBabbage** [comment]
I would send this over to Donna to check content. I will bring this up in retro as well, as I am unsure whether devs should be reviewing content. With this is mind you need to correct your quick launch as it no longer works. We now use the schema upload to gcs url so gist are not needed each time. Therefore yours would look like 

https://staging-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/nisra-csh-changes/schemas/en/census_individual_gb_nir.json

**Pull request merged**